app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/placement_group.rb

```diff
@@ -0,0 +1,2 @@
+class ManageIQ::Providers::IbmCloud::VPC::CloudManager::PlacementGroup < ::PlacementGroup
```

Comment:
I don't think it's necessary to subclass `PlacementGroup` in `IbmCloud::VPC` when it hasn't been implemented in it's inventory. (don't think it hurts anything... but may be confusing)